docs/src/examples/simple-native.md

```diff
@@ -0,0 +1,198 @@
+
+## Setup
+
+First we'll walk through the basic setup and usage of the project
+
+```bash
+➜  cd fuel-indexer/examples/simple-native/
+
+➜  tree . -I 'target/'
+.
+├── Cargo.toml
+├── README.md
+├── config.yaml
+├── contracts
+│   └── counter
+│       ├── Forc.lock
+│       ├── Forc.toml
+│       ├── out
+│       │   └── debug
+│       │       ├── counter-abi.json
+│       │       └── counter.bin
+│       └── src
+│           └── main.sw
+├── counter-rs
+│   ├── Cargo.toml
+│   └── src
+│       └── main.rs
+├── frontend
+│   └── index.html
+├── manifest.yaml
+├── schema
+│   └── counter.graphql
+└── src
+    └── main.rs
+
+10 directories, 14 files
+```
+
+With this [recommended project structure](../getting-started/fuel-indexer-project.md) you'll notice:
+- `contracts` is where our Sway smart contract `counter` is located.
+- `counter-rs` is a tiny HTTP web application that we use to pass messages to/from the `counter` contract.
+- `frontend` is unused (as there is no frontend for this small example)
+- `config.yaml` is our node config
+- `manifest.yaml` is our indexer config
+
+## Usage
+
+In this section we'll cover the exact steps needed to spin up this example.
+
+### Database setup
+
+We'll start by creating a database. This step assumes some familiarity with [creating Postgres roles and databases](https://learn.coderslang.com/0120-databases-roles-and-tables-in-postgresql/). In this example we're using an `indexer` database owned by a `postgres` role without a password.
+
+- Note that some of these commands may differ based on your local setup
+
+```bash
+createdb -U postgres indexer
+```
+
+Next we'll bootstrap our database by running some migrations. These migrations are repsonsible for creating our `graph_registry`. 
+
+- The `graph_registry` is a component that is responsible for keeping track of database columns and columns types. The `graph_registry` is largely abstracted away from the user, but can be seen by inspecting the `graph_registry` schema.
+
+```bash
+cd fuel-indexer/
+
+DATABASE_URL="postgres://postgres@127.0.0.1:5432/indexer" \
+    diesel migration list --migration-dir=schema/migrations/postgres
+```
+
+### Starting the webserver & Fuel node
+
+As previously mentioned, `counter-rs` contains both a tiny webserver used to pass messages to/from our Sway smart contract, as well as our Fuel node that we use to interact with the blockchain. We will start both of these services with the following command.
+
+```bash
+cd fuel-indexer/examples/simple-native/counter-rs
+
+RUST_LOG=debug cargo run
+```
+
+### Start the fuel indexer service
+
+With our Fuel node and webserver up and running, we'll next start our Fuel Indexer service.
+
+```bash
+cd fuel-indexer/examples/simple-native/
+
+RUST_LOG=info cargo run -- ./config.yaml ./manifest.yaml
```

Comment:
```suggestion
cargo run -- ./config.yaml ./manifest.yaml
```

I could be wrong, but I *think* `info` is the default log level?

Also, it's a shame we have to pass `./config.yaml` and `./manifest.yaml` manually, but I guess working out some sort of standard/default locations for these things could be worked out as a part of #89 or some other issue.